modules/appeals_api/spec/docs/supplemental_claims/v0_spec.rb

```diff
@@ -181,16 +182,17 @@ def swagger_doc
       consumes 'application/json'
       produces 'application/json'
 
-      parameter name: :sc_body, in: :body, schema: { '$ref' => '#/components/schemas/scCreate' }
-
-      parameter in: :body, examples: {
```

Comment:
I noticed there were two places where the body was specified twice - this works fine but I combined these calls to `parameter` into one